File: web2-Project/src/app/entities/flight-presedanje/presedanje.ts
import { StringForICollection } from '../StringForICollection/string-for-icollection';

export class Presedanje {
    id: number;
    brojPresedanja: number;
    gradoviPresedanja: Array<StringForICollection>;

    deleted: boolean;

    constructor(brojPresedanja: number, gradoviPresedanja: Array<StringForICollection>) {
        this.brojPresedanja = brojPresedanja;
        this.gradoviPresedanja = gradoviPresedanja;

        this.deleted = false;
    }
}

File: MAANPP20/MAANPP20/Common/Models/Flights/Presedanje.cs
using Common.Models.Common;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace Common.Models.Flights
{
    public class Presedanje
    {
        [Key]
        public int id { get; set; }

        public int brojPresedanja { get; set; }

        public ICollection<StringForICollection> gradoviPresedanja { get; set; }

        public bool deleted { get; set; }
    }
}
